schedule/qam/common/virt/qam_virt_migration.yaml

```diff
@@ -0,0 +1,31 @@
+name: qam_virt_migration
+description:    >
+  Install guest VMs for virtualization tests.
```

Comment:
Please also add maintainer info here -- your name <mail>, qe-virt@suse.de

Reply:
Besides, based on the description and the contents of the file, indeed this file manages the schedule for mainly the steps after host installation and before migration, how about making the name of the file `qam_virt_migration.yaml` more reflecting what it does?